textproc/most/Portfile

```diff
@@ -18,9 +18,9 @@ long_description \
 homepage        http://www.jedsoft.org/${name}/
 master_sites    https://www.jedsoft.org/releases/${name}/
 
-checksums       rmd160  46017a27d9104707c86c6c8b0e89c2f5cc30dfa8 \
-                sha256  db805d1ffad3e85890802061ac8c90e3c89e25afb184a794e03715a3ed190501 \
-                size    162172
+checksums       rmd160  2abd8b43c9e5f44f0f93d93b231b70f1a69238f2 \
+                sha256  9455aeb8f826fa8385c850dc22bf0f22cf9069b3c3423fba4bf2c6f6226d9903 \
+                size    256075
```

Comment:
Curious that the size of the tarball increased 58% in this version. That's a big increase for a patch-level update. We should compare the two tarballs to make sure this size increase makes sense.

Reply:
5.2.0 includes a 471K testfiles directory that 5.1.0 didn't have.